### PR TITLE
 added input-file-upload.md FR [skip ci]

### PR DESCRIPTION
**Pull request by jaaf**
| Q | A |
| --- | --- |
| Is bugfix? | yes/no |
| New feature? | yes/no |
| Breaks BC? | yes/no |
| Tests pass? | yes/no |
| Fixed issues | comma-separated list of tickets # fixed by the PR, if any |

### CODE REVIEW

File: docs/guide-fr/concept-components.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Composants
 ==========
 
-Les composants sont les blocs de constructions principaux de vos applications Yii. Les composants sont des instances de la classe [[yii\base\Component]],
+Les composants sont les blocs de constructions principaux de vos applications Yii. Les composants sont des instances de la classe [[yii\base\Component]](http://denentzat.fr),
```
Comment (samdark) [inline docs/guide-fr/concept-components.md]: Hmm... that's not what it should be linked to.

**Comment by jaaf** [comment]
I apologize. I was just wondering what is [[ ]] in markdown and what is the difference with [ ]( ) so I made some trial to see the result. After that I forgot to put the file back to its original state.
BTW could you tell me how the link mechanism works in this case (not url explicitly given)  and what happens if the file is used on an other site?
Thank you for finding the mistake.

**Comment by samdark** [comment]
We have our own markdown extensions on top of the GitHub-flavored one. `[[className]]` is one of those.

**Comment by samdark** [comment]
It won't work well on other websites. Will be displayed as is.

**Comment by jaaf** [comment]
Ok, I see.  Thanks.

**Pull request merged**